src/Workspaces/Core/Portable/CodeStyle/CodeStyleHelpers.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Microsoft.CodeAnalysis.CodeStyle
+{
+    internal static class CodeStyleHelpers
+    {
+        public static CodeStyleOption<bool> ParseEditorConfigCodeStyleOption(string arg)
+        {
+            var args = arg.Split(':');
+            if (args.Length != 2)
+            {
+                return CodeStyleOption<bool>.Default;
+            }
+
+            bool isEnabled = false;
+            if (!bool.TryParse(args[0], out isEnabled))
+            {
+                return CodeStyleOption<bool>.Default;
+            }
+
+            switch (args[1].Trim())
+            {
+                case "none": return new CodeStyleOption<bool>(value: isEnabled, notification: NotificationOption.None);
```

Comment:
severity=none could be ambiguous to some parsers.  maybe `:`?  

`csharp_style_inlined_variable_declaration = severity=none`

`csharp_style_inlined_variable_declaration = severity:none`